waiter/integration/waiter/kubernetes_scheduler_integration_test.clj

```diff
@@ -40,6 +40,20 @@
             (is (<= initial-rs-snapshot-version rs-snapshot-version'))
             (is (< rs-snapshot-version' rs-watch-version'))))))))
 
+; test that we can provide a custom docker image that contains /tmp/index.html with "Integration Test Image" in it
+(deftest ^:parallel ^:integration-slow test-kubernetes-custom-image
+  (testing-using-waiter-url
+    (when (using-k8s? waiter-url)
+      (let [{:keys [body]} (make-kitchen-request
+                             waiter-url
+                             {:x-waiter-name (rand-name)
+                              :x-waiter-image (System/getenv "INTEGRATION_TEST_CUSTOM_IMAGE")
+                              :x-waiter-cmd "cd /tmp && python3 -m http.server $PORT0"
```

Comment:
This seems like it will be very difficult to test with anything but the "twosigma/integration" image you built for this test. Something a little more flexible might be to use a custom environment variable set to some sentinel value (e.g., `WAITER_CUSTOM_IMAGE_SENTINEL='Integration Test Image'`), which is used in the custom command. The assertion at the end of this test would compare with the sentinel value.